server/openapi_server/controllers/tool_controller.py

```diff
@@ -18,7 +18,7 @@ def get_tool():  # noqa: E501
         version="1.0.0",
```

Comment:
Remember to bump this value before making a new release. Let me know before you want to create a new release and I will show you a nice way to make them so that the changelog is automatically generated.

Reply:
Okay, after merging this PR, let's make a branch to bump the version and make a release right after that